tests/func/test_repro.py

```diff
@@ -405,7 +407,17 @@ def test(self):
             self.assertEqual(ret, 0)
 
         ret = main(
-            ["run", "--no-exec", "--single-stage", "-f", DVC_FILE] + deps
+            [
+                "run",
+                "--no-exec",
+                "--single-stage",
+                "-f",
+                DVC_FILE,
+                *deps,
+                "ls {}".format(
+                    " ".join(dep for i, dep in enumerate(deps) if i % 2)
```

Comment:
On above code, `deps` are saved as list with sequence of `-d, dep` items (flattened out), i just extract every second element of the list.